canban-react/src/services/headerForAddCards.js

```diff
@@ -0,0 +1,34 @@
+import {url} from './serverConfig.js';
+
+
+export async function headerForAddCards(event){
+	if(event.target.className==='addButton'){
+  		let text = prompt("Eneter text for new card");
+  		if(text === null || text===''){
+  			return;
+  		}
+
+  		
+  		let saveToStorage = createNewArrCardObj(text, event.target.closest('.columns').id);
+
+  		let addNewCardObj = async()=> {
+  			let response = await fetch(url + 'api/card', 
```

Comment:
don't use fetch directly in component, you should use something like actions layer for this